### ISSUE TITLE
Investigate other uses for GNN model to other bioentities/questions

### ISSUE THREAD

**Issue by dkoslicki** [RTXteam/RTX#1178]
Investigate additional applications of explainable graph neural networks to other classes of bioentities other than just drugs and diseases (eg. phenotype prediction from groups of proteins, polypharmaceutical adverse events, etc.) 

- [ ] Come up more such example applications (other link predictions problems, node label prediction, etc.)
- [ ] Find associated training data (this is the hard part)

Could possibly use predicates in KG2 (along with their antonyms) to generate TP/TN training data.

**Comment by dkoslicki** [comment]
Let's back burner/close this as we have the xCRG part done, and other extensions of xGNN are more "personal project" than sponsor requested efforts
